internal/attestors/vsa/vsa.go

```diff
@@ -115,9 +113,17 @@ func collectAttestations(ctx context.Context, opts *config.VsaCommandOptions) ([
 }
 
 func evaluatePolicy(ctx context.Context, opts *config.VsaCommandOptions, attestations []oci.Signature) (bool, error) {
-	err := downloadOPABundle(ctx, opts)
-	if err != nil {
-		return false, err
+	var bundleFilepath string
+
+	if opts.PolicyUrl.IsAbs() {
```

Comment:
How does this behave with [`file://`](https://en.wikipedia.org/wiki/File_URI_scheme)?

Reply:
https://pkg.go.dev/net/url#URL.IsAbs

Reply:
I wanted to only focus on http and local file support right now. I could add some more checks around this. I want relative paths to work as well, mostly for local testing with the policy repo, I don't think `file://` supports relative paths.